spk/gateone/src/installer.sh

```diff
@@ -35,13 +36,16 @@ postinst ()
     adduser -h ${INSTALL_DIR}/var -g "${DNAME} User" -G ${GROUP} -s /bin/sh -S -D ${USER}
 
     # Create a Python virtualenv
-    ${VIRTUALENV} --system-site-packages ${INSTALL_DIR}/env > /dev/null
+    ${VIRTUALENV} --system-site-packages ${APP_ENV} > /dev/null
 
     # Install the bundle
-    ${INSTALL_DIR}/env/bin/pip install --no-index -U ${INSTALL_DIR}/share/requirements.pybundle > /dev/null
+    ${APP_ENV}/bin/pip install --no-index -U ${INSTALL_DIR}/share/requirements.pybundle > /dev/null
 
     # Install GateOne
-    ${PYTHON} ${INSTALL_DIR}/share/GateOne/setup.py install --prefix=${INSTALL_DIR} > /dev/null
+    PYTHONPATH="${APP_ENV}/lib/python2.7/site-packages/:${PYTHONPATH}" ${PYTHON} ${INSTALL_DIR}/share/GateOne/setup.py install --prefix=${APP_ENV} --skip_init_scripts > /dev/null
+
+    # install initial certificats
+    cp /usr/syno/etc/ssl/ssl.crt/server.crt /usr/syno/etc/ssl/ssl.key/server.key ${INSTALL_DIR}/ssl/
```

Comment:
yea i realized the issue is due to the cert and key having 400 as the perms. So it must be copied over to the gateone install dir so that we can `chown` the files to gateone.